client/src/components/ManageCustomer.js

```diff
@@ -11,50 +23,53 @@ const ManageCustomer = () => {
     history.push("/modify-a-customer")
     };
 
-    return (
+    const CustomerData = () => {   
+        const {loading, error, data } = useQuery(MANAGE_CUSTOMERS);
+    
+        if(loading) return <p>Loading...</p>;
+        if(error) return <p>Error :(</p>
```

Comment:
missing a `;` at the end

Reply:
Ok, I just added this semicolon, thanks for noticing that.